views/components/ConfigurationsList/index.jsx

```diff
@@ -382,31 +245,17 @@ const ConfigurationsList = ({ user, classes, theme }) => {
       }
     })
   }
-  const removeConfig = (configs, index, configID) => {
-    removeConfiguration(configID)
-    setState((prevState) => {
-      return {
-        ...prevState,
-        configurations: update(configs, {
-          $splice: [[index, 1]],
-        }),
-        snackTime: true,
-      }
-    })
-    if (index == state.preferences['config']) {
-      updateUserPreferences(0, 'index')
-    }
-  }
-  const openSearchUsers = (index, configID, shared, owner) => {
-    setState((prevState) => {
+
+  const openSearchUsers = (config) => {
+    const { _id, readers, owner } = config
+    setSharedWith((prevState) => {
       return {
         ...prevState,
         searchUsers: true,
         selectedConfig: {
-          _id: configID,
-          index: index,
+          _id,
         },
```

Comment:
should we not set the `selectedConfig` to `config`?

Reply:
this is functionality used by the shared component and out of scope for this ticket